cli/cmd/plugin/ui/web/tanzu-ui/src/shared/constants/NavRoutes.constants.tsx

```diff
@@ -11,7 +11,7 @@ export const NavRoutes = {
     AWS: 'aws',
 
     // temp routes to be refactored out
-    DEPLOY_PROGRESS: 'progress',
+    DEPLOY_PROGRESS: '/progress',
```

Comment:
Please modify usage in ClusterAttributeStep, line 45 (navigateToProgress()), where the code is prepending '/' to that route

Reply:
fixed